tools/ci_build/github/azure-pipelines/templates/c-api-cpu.yml

```diff
@@ -350,6 +363,14 @@ stages:
         SpecificArtifact: ${{ parameters.specificArtifact }}
         BuildId: ${{ parameters.BuildId }}
 
+    #- template: flex-downloadPipelineArtifact.yml
+    #  parameters:
+    #    StepName: 'Download Android-QNN-aar Pipeline Artifact'
```

Comment:
When will we enable this? 

Would be good to add a comment explaining why it's currently commented out.

Reply:
I was waiting to hear back from the ESRP folks about security requirements for new packages before publishing the build artifacts. But looking over their docs, it seems we already have all the signing process in place so atleast for publishing to Azure we should be okay. I will add the publishing part also in this PR.